java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehavior.java

```diff
@@ -76,6 +78,17 @@ public void createYield(ExplodedGraph.Node node) {
     yields.add(yield);
   }
 
+  public void createExceptionalCheckBasedYield(ExplodedGraph.Node node, Type exceptionType, SECheck check) {
```

Comment:
The same way as method `createYield` is tested using unit test of method yield creation, through using SE on test files, I'm testing it in `ExceptionalCheckBasedYieldTest.creation_of_new_yield()`.